.github/workflows/rollback.yml

```diff
@@ -0,0 +1,304 @@
+          - type: cn
+            regions: "cn-north-1 cn-northwest-1"
+    steps:
+      - name: Configure AWS Credentials
+        if: matrix.type == 'global'
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.RELEASE_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.RELEASE_SECRET }}
+          aws-region: us-west-2
+      - name: Configure AWS Credentials for HKG
+        if: matrix.type == 'hkg'
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.RELEASE_HKG_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.RELEASE_HKG_SECRET }}
+          aws-region: ap-east-1
+      - name: Configure AWS Credentials for BAH
+        if: matrix.type == 'bah'
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.RELEASE_BAH_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.RELEASE_BAH_SECRET }}
+          aws-region: me-south-1
+      - name: Configure AWS Credentials for CPT
+        if: matrix.type == 'cpt'
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.RELEASE_CPT_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.RELEASE_CPT_SECRET }}
+          aws-region: af-south-1
+      - name: Configure AWS Credentials for MXP
+        if: matrix.type == 'mxp'
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.RELEASE_MXP_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.RELEASE_MXP_SECRET }}
+          aws-region: eu-south-1
+      - name: Configure AWS Credentials for CN
+        if: matrix.type == 'cn'
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.RELEASE_CN_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.RELEASE_CN_SECRET }}
+          aws-region: cn-north-1
+      - name: Check version exists in SSM
+        run: |
+          mkdir -p ${{ env.SSM_ARTIFACT_ROOT }}
+          for region in ${{ matrix.regions }}; do
+            document_versions=$(aws ssm list-document-versions --name ${SSM_RELEASE_PACKAGE_NAME} --region ${region})
+            document_version=$(echo $document_versions | docker run --rm -i stedolan/jq -c -r ".DocumentVersions|.[]|select(.VersionName==\"${{ github.event.inputs.rollback_version }}\")|.DocumentVersion")
+            if [[ -z $document_version ]]; then
+              echo "Document version not found for ${{ github.event.inputs.rollback_version }} in $region"
+              exit 1;
+            else
+              echo "$region $document_version" >> ${{ env.SSM_ARTIFACT_ROOT }}/versions.txt
+            fi
+          done
+      - name: Upload versions artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ matrix.type }}-versions
+          path: ${{ env.SSM_ARTIFACT_ROOT }}
+  all-checked:
```

Comment:
IMO should we leave this since these conditions and steps can be run on ```image-rollback``` jobs though?